controller/WebController.php

```diff
@@ -329,23 +329,47 @@ public function invokeGlobalSearch($request)
         $vocabObjects = array();
         if ($vocids) {
             foreach($vocids as $vocid) {
-                $vocabObjects[] = $this->model->getVocabulary($vocid);
+                try {
+                    $vocabObjects[] = $this->model->getVocabulary($vocid);
+                } catch (Exception $e) {
+                    // skip vocabularies not found in configuration
+                    // please note that this may result in global search
+                    // NB: should not happen in normal UI interaction
+                }
             }
         }
         $parameters->setVocabularies($vocabObjects);
 
+        $nondefaultEndpointVocs = array();
```

Comment:
The idea here is to report to the user (via `skipped_vocabs` passed to Twig template) that actually not all vocabularies were searched in the global search, since only the default SPARQL endpoint is used in global search, but not other vocabulary-specific endpoints.

Reply:
Might be better to fail fast here when any vocabulary ID is not known. At least then we wouldn't accidentally trigger global search.

Reply:
Switched to fail fast operation.